src/client/components/CreateSeason.jsx

```diff
@@ -25,23 +36,68 @@ export const CreateSeason = () => {
         if(completedForm) postForm()
     }, [formStep])
 
-    const postForm = () => {
+    const [competitors, setCompetitors] = useState([])
+
+    useEffect(async() => {
+        const token = localStorage.getItem(LOCAL_STORAGE.TOKEN);
+        setToken(token)
+
+        const getCompetitions = async () => {
+            const response = await fetch(API_URL.COMPETITION_GET, {
+                method: HTTP_METHOD.GET,
+                headers: {
+                    Authorization: token,
+                },
+            });
+            const results = await response.json();
+            return results.data[0].competitors;
+        };
+
+        const competitors = await getCompetitions();
+        setCompetitors([...competitors])
+    }, [])
+
+    const postForm = async() => {
         console.log(form)
-    }
+        const fetchConfig = {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+                Authorization: token,
```

Comment:
I'm not going to let this stop me approving the PR, but I _think_ it's good practice to include the string "Bearer" here, see: https://security.stackexchange.com/questions/108662/why-is-bearer-required-before-the-token-in-authorization-header-in-a-http-re - i.e:
```
Authorization: `Bearer ${token}`,
```

Reply:
Bearer is already included in the token when it is created
![image](https://user-images.githubusercontent.com/90713858/158800241-dadcc0c4-0e63-4bf9-82a1-45d694426968.png)